CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Changelog
+# Version 3.65.0 (2024-03-04)
```

Comment:
since 3.64.0 has never been released, can we just rename the release notes 3.64.0 -> 3.64.1 ?

Reply:
I asked in channel, Matt Roberson and Adrian both suggested a new version
https://labelbox.slack.com/archives/C06GDL07QSK/p1709572943101949